src/components/user-card/UserCard.js

```diff
@@ -1,10 +1,32 @@
 import React from 'react';
 
 //todo здесь нужно сделать импорт стилей из './UserCard.scss'
+import './UserCard.scss';
 
 
 // todo здесь нужно сделать экспорт функии под названием UserCard с аргументом props
+
+export const UserCard = (props) => {
+    const {user} = props;
+    if(!user) return null;
+    const {first_name, last_name, email, address, _links:{avatar:{href}}} = user;
+    return(
+
```

Comment:
This empty line should be deleted
and before return there could be empty line for code's visual separation